edk2toolext/tests/test_pyopenssl_signer.py

```diff
@@ -6,8 +6,166 @@
+p2m2/AVJ2aejS5fZHOgRXOb5YeO3TyyG36bsAy6Le7ZqcLEtESd4CbjCKJ1i\
+mcGdFk18Jh3JN/lqefpB3W3l9YRxL2Cu8jzarlpWQw6O7Q2CjLbZ5T+kSZqK\
+EByQLViCvCILYyTB8Kr5go/GGuU2oZYC1dWprxGDhznMO/KewNXec42h7j6K\
+rIS6ftJFMLKYBsfsZ8Q3MIvCcPNz/ch7lymzibtjYJE8VwJlcL/RP17tOi4J\
+hC44gdErhUjj87s3kh6ZDY3FToU7u/voQd/1ipRTZCurOFc20RbhDfyOyafv\
+SyASl7Hlk3+sGVGHvtrSBoSYsW+4XNyOfqnZhp+EzrBmg9AjfvJniWtMTbWI\
+ekO/DbjVxXww+/AXkTgqcutjFerTWj16fKy7wqM3tCVQiTsbX7uiWDF6nEF6\
+NQAZ1EsUaEw2B45eUh8xtML3j70zFQRwIJ+qnPxuj3ouWk4ShZKjQD39EsJn\
+5aVzGWVIF9tPgJo10E1vf+doIHmpgpzYHJi+fLPJZ6bbrzpntK9vcnUjnsK1\
+yRax/+381jhIZbAG54ruOnANfUhOWijObTdE5we2g9j3JpzXWUYoFUBFqBx4\
+hYAwgFw//fp3b8VtI8xUnzrvfBCF4rUyykWT7YnUM8VuBFgq5GGPUt1PZaw/\
+jemPMV5WNe+ReL/BKjniZH5ZwITbz4PHm6Qt/G5tM/h46YgRux5jy7KriPQw\
+PqgoJK8JFs8I/6mQkv2fNY6ttbRMbSdi0yLPpqgQk8EEJV8mvI/bwXGeJTLQ\
+JhXCau48DBR9nYcyaYCDX3z84yZpVJFRJyWT9GvTMbA3zbS3BdO1oaC/60No\
+1faabkgJyzXeNmY/v293JI0sSLB51tQr2amSH1mu2kvvhMegs2qw57Kg8r3i\
+W9tjo7GCepbUbXa9BE09ZDRsuOuNWfs/2Pe6X6P2Hg3gc7+aaspDCCBUEGCS\
+qGSIb3DQEHAaCCBTIEggUuMIIFKjCCBSYGCyqGSIb3DQEMCgECoIIE7jCCBO\
+owHAYKKoZIhvcNAQwBAzAOBAjHl66YnVMBEwICCAAEggTIpiydhQsoj1647s\
+pVJEC07g1PcNCGS985FvjILNgvEm8uZe2qPWK5wVhN81vQ6lx65q7DXqEncx\
+2UdzidAxgDxVAHBBqHhLXkDMItng9/Q4rVS9E5IPsOVrtl+0tXsYvcktE/1T\
+dVoNzClYNrwBE82Nc9E+fDBEQ6MLF7pnF1IgfVgMOHOkn4GT8XgZ9Y8kg6us\
+/PpexIQwayRnIa5hmVAlVKYUuW9WZR782PDFPU+DwwIe+LMwmDubGxWDqBID\
+hhPG9vhdWqb7U5wl33ryLAs1/SMC1J9AbOUQIJvTJfnFd/hfg35lq6lCiEFZ\
+S+dPdAXqVOOgdHlusTskeYlol5EZxPATDYHm5t7PaiQ5PtfRQN1G2T8yB/eo\
+/NbA4l1vzscTHybqucb8AAqf0WdsPXo+VT4FpK8SBM/C5TANRbuxEm9rNnKN\
+I0VIZV+4etGHmi8OW5y65SysUn/D7RL1atHEEPcMesfWiks7aBZLwYJCpO5p\
+Ry5n7T3Ca/E19GSubnYSsXAmXfBJ+lw8jq99yjPML+jM/lGzv300ajpzjDtI\
+xyTfbaO0IH081RfDmmer1dkd5C1RaOzswRIga7KnXjyJz/f4Yh4tCdKsQN44\
+PE6ib+dvZWED9vX7Zh8IqdM7bdZB6Lj8ZDQD2nZ/7rkOJpqR6qisTaAUdnmw\
+DN0UJ9F8OxEkS6YodI9a/xjmtxFGbDVCow0R01zBiK+letgrr5iPnXEadiFd\
++E8zcCl5Qkk+gP8PqphOGLZZgEkkeLETk82w9D0XuBHc2grXlcNUvRBm1D8j\
+VAIIrIogvNJL/itjHAoZEjeE8n0RQYmN2cWMJlNk0Ifhc17Ycv9O09QO4RY6\
+OOMRaFtRSOJScrxTPVHSyceSWs85HV11BYzYYZQd0xHy1+nz2ZKyrm7VXsWf\
+qIWE/HkyspodjUUNBV2iPJR6UHxloS5EGpluG+ApyXMh/efjRZ9isLUp9lEA\
+n8HghTJwi/vpBwWot4nGrg6RpHKrmhxAi564rVHuJrhDCqQvAx1atO8zmCqJ\
+Ppt7kYm3ll6k4HZPetlQzqfGLedS4a4OHb+3c4VaqONUTZDAE0cQ4KeR/byf\
+Z+UIDGAosmEWk49CnrPF/eD3Ul79yBNqOZIg3P6V+pwek9nYX8fCWWRk4Ayc\
+SQjHY0WdTCUUvc14TwO5Lc/AZSrnCG5KB/YuIhgQlLKMRioQ4uS829LNYAhz\
+8xSRG9MRZjxIdUevqxDEI1/eKk2+qBnq+qVqoltPw+8mqnsMM3qRMtBZydLV\
+s5pMklXuIMGVe2PGLwVY2cbc4ms4m2PpyLBO0DqlZJ5G8hJuq11SE8jjfI3K\
+O9xc6D28mryoSTJE3ruN3mlRdbiAsNg15mUdXq9M9AthLaeSyCLT3a2QP1zt\
+O+AHsgd6ovMhfxAZ1R/0YayjtKB6Z4CN5rEAg2QdR5nvb6G4EKyqWAZhCCcd\
+UezFsMna+YaA/F2o9WIj9KPB+rdNh/KcbIJhpA2DSbbTPejHAk1TYBWall3B\
+2CfsFEIMjtuid/ggz/pUYCb88PN3BPoQ7GO/jv6Vi6F4oCQz+Y0srbmIStNj\
+qKXmc5OHIJoRkbrbFi4+2BeevncDADGrhl3heGWSmRlbOPxrryNrDPw7cbMS\
+UwIwYJKoZIhvcNAQkVMRYEFILT9pNixD3s66GdK3I48b/dr23DMDEwITAJBg\
+UrDgMCGgUABBTdiDC3a0y1gAbO1eZqveI3Zd0BrwQIFO8dSTjGLhMCAggA'
+
+# TEST_CERT2 has a password
```

Comment:
Update TEST_CERT2 and TESTCERT2 to match.  Please do same for TEST_CERT1

Reply:
Updated